sci-electronics/kicad/metadata.xml

```diff
@@ -1,8 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<herd>proxy-maintainers</herd>
+	<use>
```

Comment:
it's absolutely not clear what the python USE flag does for this package, so it must be documented locally